docs/persistence.rst

```diff
@@ -449,7 +449,7 @@ Potential workflow for a set of time based indices governed by a single template
             return super().save(**kwargs)
 
     # once, as part of application setup, during deploy/migrations:
-    logs = Log._index.as_template()
+    logs = Log._index.as_template('logs-*')
```

Comment:
Thank you for the patch, good catch!

Could I just ask you to make this `logs` as template names shouldn't include wildcards. Thanks!